index.php

```diff
@@ -59,6 +73,24 @@
     }
     header( 'Location: index.php');
   }
+
+  if ( isset( $_POST['amq'] ) && $_POST['amq'] == "send" ) {
+    $amq_result = "";
+    $stomp = new Client('tcp://broker-amq-tcp:61613');
+    $stomp->setLogin($AMQ_USER, $AMQ_PASS);
+    try {
+      $stomp->connect();
+      $message = trim($_POST['message']);
+      $stomp->send('/queue/inventoryReceived', $message);
+      $amq_result = "success";
+    } catch (StompException $e) {
+      die("Failed to send message: " . $e);
+    }
+    $stomp->disconnect();
```

Comment:
does this need to be in a `finally` block?

Reply:
Well, sounds like a good practice to me.